python/mpi_cmake_modules/clang_format.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
As this is an executable script, it should rather be placed in `/scripts`.

Reply:
this has an API and can be used as main. I need to put it as part of the python API in case someone wants to base another executable on it, typically the clang-format-diff or check that can be added.

The fact that it is flagged as executable allow the symlink to be used by the rosrun

Reply:
I think it would be cleaner to separate the API part and the executable script.  Then the symlink would also not be needed.  But I guess this is just a matter of taste, I'm also okay with keeping it as is.